.appveyor.yml

```diff
@@ -1,40 +1,31 @@
-image:
-  - Visual Studio 2017
-  - Ubuntu
```

Comment:
Can we keep Ubuntu in the matrix?

Reply:
I was thinking you might be interested in leaving linux/macOS to travis and windows to appveyor. But I can add it back.

Reply:
I was going to say that someone added Ubuntu support just recently, so I would not want to waste their effort, but apparently that someone was [you](#261), so I'll leave this for you to decide :).

Reply:
Actually there is still a use for it that Travis doesn't cover. I'll add it back after the VS2013 thing gets figured out.